src/modules/launcher/Plugins/Microsoft.Plugin.Program/Programs/Win32Program.cs

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) Microsoft Corporation
```

Comment:
This was a UTF8 files, but the other files in this project are UTF8 + BOM, seems like a sensible change to add?